packages/react-reconciler/src/ReactDebugFiberPerf.js

```diff
@@ -318,7 +318,8 @@ export function stopPhaseTimer(): void {
   }
 }
 
-export function startWorkLoopTimer(): void {
+export function startWorkLoopTimer(nextUnitOfWork: Fiber | null): void {
+  currentFiber = nextUnitOfWork;
```

Comment:
This should be inside the `if (enableUserTimingAPI)` check.

Reply:
Updated